### DISCUSSION TITLE
ScrollablePanel, empty items

### DISCUSSION THREAD

**Discussion by Nesquik** [rexrainbow/phaser3-rex-notes#225] [Q&A] [answered]
Hi,
I'm in a bit of a pickle trying to integrate this demo https://codepen.io/rexrainbow/pen/YMyBom into my project.
I'm using the following template: https://github.com/yandeu/phaser-project-template

Not sure where I'm going wrong but I'm not getting the proper render of the components themselves.

Here's a repro of the issue on glitch: https://glitch.com/edit/#!/wave-lavender-squid?path=src%2Fscripts%2Fobjects%2FscrollablePanel.ts%3A1%3A0

Thanks for the help!

**Comment by rexrainbow** [comment, marked as answer]
`new RoundRectangle(...)` creates game object, but it does not add to scene. Thus it won't be displayed.
Use `scene.add.existing(rect)` to add it to scene.

[Reference](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/shape-roundrectangle/#import-class)
